src/web/pages/base.ts

```diff
@@ -107,38 +107,44 @@ class Page {
     return protractor.browser.switchTo().frame(this.getElement(elementName).getWebElement());
   }
 
-  public waitForVisibilityOf(elementName: string) {
+  public waitForVisibilityOf(elementName: string | ElementFinder) {
     return waitForVisibilityOf(this.getElement(elementName));
   }
 
   public waitForInvisibilityOf(elementName: string) {
     return waitForInvisibilityOf(this.getElement(elementName));
   }
 
-  public getElement(elementName: string) {
-    if (!this[elementName]) {
-      console.warn(
-        chalk.grey(
-          `Element "${elementName}" does not exist in the currentPage. CSS selector will be build from the string!`
-        )
-      );
-      return element(by.css(elementName));
+  public getElement(elementName: string | ElementFinder) {
+    if (!this[String(elementName)]) {
+      if (elementName instanceof protractor.ElementFinder === true) {
```

Comment:
1. I think that you don't have to use `protractor. ElementFinder`, you can just use `ElementFinder` as you imported it from protractor
2. I think it's enough to `elementName instanceof protractor.ElementFinder`

Reply:
to point 1. - I have to use "protractor.ElementFinder". If I use only"ElementFinder"  I have issue "RangeError: Maximum call stack size exceeded" and PromiseRejectCallback can't go out from the loop. We talked about this, do you remember?